CONST_vars.yaml

```diff
@@ -231,7 +235,7 @@ vars:
     # Checker configuration
     checker:
         fulltextsearch: text to search
-        lang_files: [ngeo, cgxp, cgxp-api]
+        lang_files: [cgxp, cgxp-api]
```

Comment:
I probably need to keep `ngeo` here. @sbrunner, do you think that you should be backported to c2cgeoportal?